### PR TITLE
Only require simplecov if RUBY_ENGINE=="ruby"

### PR DESCRIPTION
**Pull request by plantfansam**
Shameless rip of https://github.com/thoughtbot/factory_bot/pull/1563/files. Alternate attempt to resolve the same thing as #1422. Let's see how CI goes...

### CODE REVIEW

**Comment by arielvalentin** [comment]
@plantfansam I think you _have_ to wrap the require and usage in the engine check otherwise you will get a `NameError`

File: test_helpers/test/test_helper.rb [hunk 1]
```diff
@@ -9,6 +9,6 @@
 require 'minitest/autorun'
 require 'pry'
 
-require 'simplecov'
+require 'simplecov' if RUBY_ENGINE == "ruby"
 SimpleCov.start
 SimpleCov.minimum_coverage 85
```
Comment (arielvalentin) [inline test_helpers/test/test_helper.rb:14]:
```suggestion
 if RUBY_ENGINE == "ruby"
  require 'simplecov'
  SimpleCov.start
  SimpleCov.minimum_coverage 85
end
```

Like this no?

Comment (plantfansam) [reply test_helpers/test/test_helper.rb:14]: yeah you're right - let me update

**Review by arielvalentin — APPROVED** [review]

**Comment by arielvalentin** [comment]
good stuff @plantfansam

**Comment by arielvalentin** [comment]
@plantfansam doah! Looks like there were a few spots you missed.

**Comment by plantfansam** [comment]
> @plantfansam doah! Looks like there were a few spots you missed.

not my finest hour LOL. Will thrash on this pr and ping you when tests are green.

**Comment by arielvalentin** [comment]
> not my finest hour LOL. Will thrash on this pr and ping you when tests are green.
What does it say about me that I approved the PR before the builds finished?

**Comment by plantfansam** [comment]
@arielvalentin specs are green! 🎉  /cc @fbogsany

**Review by arielvalentin — APPROVED** [review]

**Review by fbogsany — APPROVED** [review]

**Pull request merged**